src/lang/lt.js

```diff
@@ -0,0 +1,8 @@
+const lt = (value, other) => {
+  if (value < other){
+    return true;
+  }
+    return false;
+}
+
```

Comment:
If you see this pattern, you guys could also make it a one liner doing:
```
return (value < other) 
```

a check with a boolean operator will return a boolean so you can just return the return of the result of the boolean expression